src/utils/error-messages.js

```diff
@@ -13,8 +13,8 @@ exports.CONFIGURATION = {
   AUTH_SECRET_MISSING: 'Your Forest authSecret seems to be missing. Can you check that you properly set a Forest authSecret in the Forest initializer?',
 };
 
-exports.IP_WHITELIST = {
-  SERVER_DOWN: 'Cannot retrieve the list of whitelisted IPs. Forest API seems to be down right now.',
-  SECRET_NOT_FOUND: 'Cannot retrieve the list of whitelisted IPs. Can you check that you properly copied the Forest envSecret in the Liana initializer?',
-  UNEXPECTED: 'Cannot retrieve the list of whitelisted IPs. An error occured in Forest API.',
+exports.SERVER_TRANSACTION = {
+  SERVER_DOWN: 'Cannot retrieve the data from the Forest server. Forest API seems to be down right now.',
+  SECRET_NOT_FOUND: 'Cannot retrieve the data from the Forest server. Can you check that you properly copied the Forest envSecret in the Liana initializer?',
+  UNEXPECTED: 'Cannot retrieve the data from the Forest server. An error occured in Forest API.',
```

Comment:
It is still the case, I concat more info with this error. Since the service which make the request is more generic, the error must be too. Then it is the responsability of who use the service to improve the error.

Reply:
VError added